src/ParallelAlgorithms/Interpolation/Protocols/ComputeVarsToInterpolate.hpp

```diff
@@ -0,0 +1,121 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <type_traits>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace Frame {
+struct ElementLogical;
+struct Grid;
+struct Inertial;
+}  // namespace Frame
+template <size_t Dim>
+struct Mesh;
+struct DataVector;
+/// \endcond
+
+namespace intrp::protocols {
+/*!
+ * \brief A protocol for the type alias `compute_vars_to_interpolate` that can
+ * potentially be found in an InterpolationTargetTag (potentially because an
+ * InterpolationTargetTag does not require this type alias).
+ *
+ * \details A struct conforming to the `ComputeVarsToInterpolate` protocol must
+ * have
+ *
+ * - a type alias `allowed_src_tags` which is a `tmpl::list` of tags from the
+ *   volume that can be used to compute quantities that will be interpolated
+ *   onto the target.
+ *
+ * - a type alias `required_src_tags` which is a `tmpl::list` of tags from the
+ *   volume that are necessary to compute quantities that will be interpolated
+ *   onto the target. This list must be a subset of `allowed_src_tags`.
+ *
+ * - a type alias `allowed_dest_tags<Frame>` which is a `tmpl::list` of tags on
+ *   the target that can be computed from the source tags.
+ *
+ * - a type alias `required_dest_tags<Frame>` which is a `tmpl::list` of tags on
+ *   the target that must be computed from the source tags. This list must be a
+ *   subset of `allowed_dest_tags<Frame>`
+ *
+ * - an `apply` function with at least one of the signatures in the example.
+ *   This apply function fills the `vars_to_interpolate_to_target` type alias
+ *   from the InterpolationTargetTag. Here `Dim` is `Metavariables::volume_dim`,
+ *   SrcFrame is the frame of `Metavariables::interpolator_source_vars` and
+ *   TargetFrame is the frame of `vars_to_interpolate_to_target` from the
+ *   InterpolationTargetTag. The overload without Jacobians treats the case in
+ *   which TargetFrame is the same as SrcFrame. Used only *with* the
+ *   Interpolator ParallelComponent.
```

Comment:
ComputeVarsToInterpolate will also be used without the Interpolator ParallelComponent after #3828

Reply:
Should I change the docs now? Or should I wait to see which is merged first?

Reply:
Common practice is that the PR that get merged second does the rebase. In this case it wouldn't hurt to just delete the sentence in this PR already, while you're updating the comment below.